.github/workflows/molecule.yml

```diff
@@ -94,10 +93,14 @@ jobs:
         with:
           path: "${{ github.repository }}"
       - name: molecule
-        uses: robertdebock/molecule-action@4.0.9
+        uses: gofrolist/molecule-action@v2
```

Comment:
Why switching to gofrolist/molecule-action?

Reply:
No real reason, I just following the reference of what @bngameni did here while updating to molecule v5: https://github.com/claranet/ansible-role-nodejs/blob/01a94c33716df8b216c053ce580047cae74c8ec0/.github/workflows/molecule.yml#L71